crates/swc_plugin/Cargo.toml

```diff
@@ -1,19 +1,14 @@
 [package]
 authors = ["강동윤 <kdy1997.dev@gmail.com>"]
-description = "Base for swc plugins"
+description = "SDK for authoring swc plugin"
 documentation = "https://rustdoc.swc.rs/swc_plugin/"
-edition = "2021"
+edition = "2018"
 license = "Apache-2.0"
 name = "swc_plugin"
 repository = "https://github.com/swc-project/swc.git"
 version = "0.24.0"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
```

Comment:
We don't re-export plugin macro from `swc_plugin`?

Reply:
Good point, I think I just forgot it. Updated.